javascript/javascript3/week3/art.js

```diff
@@ -0,0 +1,88 @@
+const canvas = document.getElementById("myCanvas");
+const context = canvas.getContext("2d");
+let canvasPosition = getPosition(canvas);
+
+class Circle {
+    constructor(x, y, r, startAngle, endAngle, fillColor, context) {
+        this.x = x;
+        this.y = y;
+        this.r = r;
+        this.startAngle = startAngle;
+        this.endAngle = endAngle;
+        this.fillColor = fillColor;
+        this.context = context;
+    }
+
+    draw() {
+        this.context.beginPath();
+        this.context.arc(this.x, this.y, this.r, this.startAngle, this.endAngle, false);
+        this.context.fillStyle = this.fillColor;
+        this.context.fill();
+        this.context.stroke();
+    }
+
+}
+
+const x = Math.round(Math.random() * 100);
+console.log(x);
+const y = Math.round(Math.random() * 80);
+console.log(y);
+const r = Math.round(Math.random() * 60);
```

Comment:
You're using Math.round(Math.random() * x) a lot => should make it a separate function.